api/application.go

```diff
@@ -9,9 +11,49 @@ var EndpointApplications = Endpoint + "applications/"
 
 // https://discord.com/developers/docs/interactions/slash-commands#create-global-application-command-json-params
 type CreateCommandData struct {
-	Name        string                  `json:"name"`
-	Description string                  `json:"description"`
-	Options     []discord.CommandOption `json:"options"`
+	Name                string                  `json:"name"`
+	Description         string                  `json:"description"`
+	Options             []discord.CommandOption `json:"options"`
```

Comment:
```suggestion
	Options             []discord.CommandOption `json:"options,omitempty"`
```

This field is optional, so this field should have `,omitempty` in the tag.